homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -312,6 +336,142 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
     MODEL_AIRFRESH_VA2: AIRFRESH_SENSORS,
 }
 
+VACUUM_SENSORS = {
+    ATTR_DND_START: XiaomiMiioSensorDescription(
+        key=ATTR_DND_START,
+        icon="mdi:minus-circle-off",
+        name="DnD Start",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        parent_key=KEY_VACUUM_DND_STATUS,
+        entity_registry_enabled_default=False,
+    ),
+    ATTR_DND_END: XiaomiMiioSensorDescription(
+        key=ATTR_DND_END,
+        icon="mdi:minus-circle-off",
+        name="DnD End",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        parent_key=KEY_VACUUM_DND_STATUS,
+        entity_registry_enabled_default=False,
+    ),
+    ATTR_LAST_CLEAN_START: XiaomiMiioSensorDescription(
+        key=ATTR_LAST_CLEAN_START,
+        icon="mdi:clock-time-twelve",
+        name="Last Clean Start",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        parent_key=KEY_VACUUM_LAST_CLEAN_STATUS,
+        entity_registry_enabled_default=True,
```

Comment:
```suggestion
```
`True` is the default value for this in `EntityDescription`, so I think it's worth only to explicitly mark those with `entity_registry_enabled_default=False` that are not to be enabled per default.